azavea/temperate
# Paywall free trial banner

## Overview

Defaults all organizations to a 14 day free trial and shows a banner counting down days remaining.

### Demo

![screen shot 2018-02-19 at 10 59 25 am](https://user-images.githubusercontent.com/1818302/36387439-f7c63e66-1566-11e8-88b1-c22e4c36a694.png)
![screen shot 2018-02-19 at 10 58 51 am](https://user-images.githubusercontent.com/1818302/36387440-f7d79382-1566-11e8-91ec-47c5b91b1f50.png)

### Notes

I used `subscription` for the org table column name and Angular classes to avoid having multiple things named `plan` since that's already used for "Adaptation Plan".

Defers creation of additional subscription level options in Django until #604.

Defers trial warning to #601 and disabling UI when trial expires to #602. 

## Testing Instructions

`./scripts/update` then `./scripts/server`. The banner should show and state 14 days remaining. Cheat and change `Organization.created_at` via `./scripts/manage shell_plus` to see the days remaining change. Can also change the subscription level in the Org admin view and the banner should disappear.

Closes #600
